src/TextInput/TextInput.svelte

```diff
@@ -156,6 +155,7 @@
         class:bx--text-input--invalid="{invalid}"
         class:bx--text-input--warn="{warn}"
         class="{size && `bx--text-input--${size}`}"
+        {...$$restProps}
```

Comment:
`{...$$restProps}` has to go before `"class="{size && `bx--text-input--${size}`}"` to avoid overriding it.

Reply:
done